### PR TITLE
CONTRIBUTING.md: no one other than PR authors should merge generally

### LINKED ISSUE #284: update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by barter-simsum**
Resolves #284

### CODE REVIEW

File: CONTRIBUTING.md
Comment (mcevoypeter) [inline CONTRIBUTING.md:57]: "should generally be deferred to in order to complete the PR" is vaguer than it needs to be. Something like the following(as a replacement for the last sentence)  is clearer in my opinion: "Although anyone can hit the merge button, defer to the PR author to merge the PR unless the PR is an urgent hotfix that needs to be merged ASAP".

Comment (barter-simsum) [reply CONTRIBUTING.md]: done

**Review by mcevoypeter — APPROVED** [review]

**Pull request merged**
